package.json

```diff
@@ -156,6 +156,7 @@
         "pg": "^8.13.0",
         "pg-connection-string": "^2.7.0",
         "pg-structure": "^7.15.3",
+        "react-hotkeys-hook": "^4.6.1",
```

Comment:
a great discovery, thank you! I'll start using it for vCore as well!

Reply:
There are some pitfalls there. For instance the library can stop default behavior but doesn't want to stop propagation, or doesn't catch key events from input elements by default